package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "node.js_gmp2021",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon ./src/index.ts",
+    "lint": "tslint -p tsconfig.json",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/VitaliBrych333/Node.js_GMP2021.git"
+  },
+  "author": "VitaliBrych",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/VitaliBrych333/Node.js_GMP2021/issues"
+  },
+  "homepage": "https://github.com/VitaliBrych333/Node.js_GMP2021#readme",
+  "dependencies": {
+    "express": "^4.17.1",
+    "joi": "^17.4.0",
+    "nodemon": "^2.0.7",
```

Comment:
You have declared nodemon in dependencies and devDependencies

Reply:
delete